browser-test/test.browser.ts

```diff
@@ -0,0 +1,26 @@
+import * as assert from 'assert';
+
+import {request} from '../src/index';
+const port = 3000;  // should match the port defined in `webserver.ts`
```

Comment:
This port tends to get used a fair bit in examples and such.  Can you use something like 7172?